mw-webapp/src/component/table/Table.module.scss

```diff
@@ -6,29 +6,54 @@ $cell-max-width: 600px;
 .table {
   width: 100%;
   height: 100%;
-  border: $table-border;
+  border-radius: $primaryBorderRadius;
   background-color: var(--primaryBgColor);
-  border-collapse: collapse;
-}
+  border-collapse: separate;
+  border-spacing: 0;
```

Comment:
Why?

Reply:
border-radius doesn't work with "collapse" css property value:
https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius 
> "The border-radius property does not apply to table elements when border-collapse is collapse."

Since that I need to use `border-collapse: separate`, but this causes unnecessary spacing between table borders:
![image](https://github.com/tritonJS826/masters-way/assets/8752900/a99923a8-42f0-4389-ac9f-dd377e760cfe)

so I use "border-spacing: 0" to fix this issue.

P.S. I added the screenshot of rounded table to the PR description.

Reply:
Yea, right